### ISSUE TITLE
New line in "authorComments" in metadata: h5p file can't be imported

### ISSUE THREAD

**Issue by JsBergbau** [h5p/h5p-php-library#96]
If you enter text in the field "Autor comments" and hit Enter a new line is generated, like here 
![grafik](https://user-images.githubusercontent.com/37013344/101495198-8f63da80-3968-11eb-8fd9-ef87c462334d.png)

In the JSON is then 
```json
"authorComments":"Test\nTest"
```
As long as the h5p stays in your LMS no problem, but when you try to import in another sytem it fails
![grafik](https://user-images.githubusercontent.com/37013344/101495421-d8b42a00-3968-11eb-93eb-72c9b4684477.png)

I think the error is in https://github.com/h5p/h5p-php-library/blob/master/h5p.classes.php 
```
'authorComments' => '/^.{1,5000}$/',
```
When you try it here https://regex101.com/r/IS3lUT/1 it finds no match. When you remove the second line, it matches again.

I've tried in course presentation and Find the words, in both this error occurs.

**Comment by otacke** [comment]
@JsBergbau I agree, the regular expression doesn't cover this case. Would your mind creating a pull request to fix the issue?

**Comment by JsBergbau** [comment]
Done https://github.com/h5p/h5p-php-library/pull/97

**Comment by otacke** [comment]
@JsBergbau Cool! Hope the H5P core team will find time to take care of it soon.
